CHANGELOG.md

```diff
@@ -3,10 +3,23 @@
 NOTE: From this version on the frameworks changed to Dynamic instead of Static library
 
 ### Bug fixes
-- 
+- In implicit authentication flow, completion block of the original request will only get notified once with cancellation of authentication flow, or actual result of the endpoint. Previously, the original request got notified in every error that happened during the authentication flow. [DE311261]
+- Logout, and login notification are now being broadcasted correctly. Previously, the notifications were being sent at incorrect time, so SDK was representing incorrect status upon these notifications. [DE310488]
+- BLE authentication can now re-start when the authorization was denied. Previously, BLE authentication was not able to re-start saying the authorization process is in progress even after it's denied from authorizing device. [DE310490]
+- External API request along with Gateway with instance modifier now generates correct URL.  Previously, SDK was generating incorrect URL for external API request (non Gateway request) when Gateway has instance modifier. [DE310249]
+- `MASMQTTClient` can now connect to a public broker that does not require username, and password. Previously, `MASMQTTClient` was forcing to provide user credentials to all brokers. [DE306921]
+- Mobile SDK now properly validates request's response type as defined. Previously, Mobile SDK was accepting mis-match content-type for 200 response as a valid response. [US349551]
+- Mobile SDK now allows to run without enabling Keychain Sharing in Xcode. Please note that in order to estbalish SSO across multiple applications, Keychain Sharing still needs to be enabled. [US320771]
 
 ### New Features
--  
+- Mobile SDK introduces an ability to configure security configuration for external APIs (such as SSL pinning), so that Mobile SDK can securely connect to external API (other than primary Gateway). [US344780]
```

Comment:
The SDK adds the ability to securely connect to external APIs beyond the primary Gateway. This allows you to configure security for external APIs (such as SSL pinning).